rubrics/services/pom.xml

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.sakaiproject.rubrics</groupId>
+    <artifactId>rubrics-service</artifactId>
+    <version>12-SNAPSHOT</version>
+    <packaging>war</packaging>
+
+    <name>Sakai Rubrics - REST API &amp; Web Components</name>
+    <description>Sakai Native Rubrics Service - Provides a REST API and Web Components for Rubrics</description>
+
+    <parent>
```

Comment:
Having this in here means that people can't customise their Sakai build by editing `master/pom.xml` as they will also need to customise this file as well.